hack/publish_build_from_tag.sh

```diff
@@ -0,0 +1,113 @@
+# Helper tool to build bundle and catalog image and push it to the quay.io registry
+# for given tag of the operator and operands.
```

Comment:
as a followup i think you should make the tag an argument (and also allow use of a SHA instead of a tag)